### ISSUE TITLE
Where to add markers into figure by Manifold

### ISSUE THREAD

**Issue by lihuiknight** [DistrictDataLabs/yellowbrick#1118]
FYI

**Comment by bbengfort** [comment]
Hello @lihuiknight - we really appreciate you diving into the Yellowbrick API and checking things out - I hope you're finding it a useful package. Unfortunately, this issue does not contain enough information for us to address your question. I just retested both the dataset download using the sample code you provided in #1116 and the Manifold example from our documentation on a fresh install of Yellowbrick. Everything seemed to work well and I could not detect anything that might be tripping you up with the latest versions of our dependencies. 

If you have more detail about your question, please feel free to edit and reopen the issue with the following information:

- A clear and concise description of your question and what you are trying to do
- Sample code that gets you to the point where your question starts
- Any code that you've tried so far that hasn't worked yet 
- Does your question involve a specific dataset? If so, can you include it or reproduce with a YB dataset?
- What OS, Python version, and Yellowbrick version are you using?

**Comment by lihuiknight** [comment]
@bbengfort  I want to set the figure format like matplotlib, however I can find the corresponding setting on Manifold.

Specifically, I want to set markers for each class like [this  ](https://matplotlib.org/api/markers_api.html) when using Manifold function like [this tutorial](https://www.scikit-yb.org/en/latest/api/features/manifold.html#continuous-target).  

I believe the yellowbrick is very helpful but I can not find the setting for figure format.

**Comment by bbengfort** [comment]
@lihuiknight - unfortunately, there is currently no easy way to pass scatter plot parameters to the `Manifold` visualizer since the visualizer does a lot of work to determine colors based on the target type. You do have access to the matplotlib `Axes` and `Figure` referenced by the visualizer, using `viz.ax` and `viz.fig`, so potentially you could modify the scatter plot by accessing the `PathCollection` if you could search for it in the underlying Axes. Unfortunately, I don't have much advice for this route. However, there is a hack that you could take advantage of if you really needed to modify the `Manifold` functionality:

```python
from yellowbrick.features import Manifold
from yellowbrick.datasets import load_concrete


class MarkerManifold(Manifold):

    def _determine_scatter_kwargs(self, y=None):
        kwargs = super(MarkerManifold, self)._determine_scatter_kwargs(y)
        kwargs["marker"] = "P"
        return kwargs


X, y = load_concrete()
viz = MarkerManifold(manifold="isomap", n_neighbors=10)

viz.fit_transform(X, y) 
viz.show()       
```

![Figure_1](https://user-images.githubusercontent.com/745966/95875634-37ca3980-0d40-11eb-8a1a-82f7079ff509.png)

This hack takes advantage of part of Yellowbrick's private API and uses inheritance to modify specific functionality. The subclass overrides `_determine_scatter_kwargs`, a member of `ProjectionVisualizer`, which is the `Manifold` object's base class. Because this is a private API, I do not recommend relying on this beyond v1.2 as it is subject to change, and this method is probably not available on other visualizers that use scatter plots. 

Note that in #556 we did our best to determine how scatter plots could be implemented that meet both the requirements of visual diagnostics and the customization requirements of our users. We tried to strike a good middle ground with #874 - sacrificing some customization ability but providing more automatic functionality.

**Comment by lihuiknight** [comment]
@bbengfort , That's great. Thanks a lot. I think if Yellowbrick can inherit the settings in matplotlib, it will be more powerful. 

I close the issue.
